cmd/mayactl/app/command/volume_info.go

```diff
@@ -276,13 +276,13 @@ Replica Count :   {{.ReplicaCount}}
 			fmt.Println("Unable to display volume info, found error : ", err)
 		}
 		w.Flush()
-	} else {
+	} else if v.GetCASType() == string(CstorStorageEngine) {
 
 		// Converting replica count character to int
 		replicaCount, err = strconv.Atoi(v.GetReplicaCount())
 		if err != nil {
 			fmt.Println("Invalid replica count")
-			return fmt.Errorf("%s", "Invalid replica count")
+			return nil
```

Comment:
Why stopped returning error ?

Reply:
because in for cobra cli we re returning nil for every return statement